docs/research/Rackula-threat-model.md

```
@@ -0,0 +1,211 @@
+  files --> spa
+  site --> user
+  user --> nginx
+  nginx --> spa
+  nginx --> api
+  api --> data
+  ci --> ghcr
+  ghcr --> host
+  host --> nginx
+```
+
+## Assets and security objectives
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)